x-pack/metricbeat/module/gcp/vertexai_logs/data.go

```diff
@@ -0,0 +1,129 @@
+// Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+// or more contributor license agreements. Licensed under the Elastic License;
+// you may not use this file except in compliance with the Elastic License.
+
+package vertexai_logs
+
+import (
+	"crypto/sha256"
+	"encoding/hex"
+	"encoding/json"
+	"fmt"
+	"time"
+
+	"github.com/elastic/beats/v7/metricbeat/mb"
+	"github.com/elastic/elastic-agent-libs/logp"
+	"github.com/elastic/elastic-agent-libs/mapstr"
+)
+
+// VertexAILogRow represents a single row from BigQuery vertex AI logs table
+type VertexAILogRow struct {
+	Endpoint        string    `bigquery:"endpoint"`
+	DeployedModelID string    `bigquery:"deployed_model_id"`
+	LoggingTime     time.Time `bigquery:"logging_time"`
+	RequestID       float64   `bigquery:"request_id"`
```

Comment:
Shouldn't we just use `string` for request id?

Reply:
Hmmm, whats the usual practice for NUMERIC fields ?